### ISSUE TITLE
IPCameraFrameGrabber TimeUnit null-check logic is reversed in constructor

### ISSUE THREAD

**Issue by jsichi** [bytedeco/javacv#411]
I tried upgrading to 3.1.0 and hit an NPE when passing in null for the TimeUnit.  Passing in non-null will work, but then the actual values will be ignored.

```
        if (timeUnit == null) {
            this.connectionTimeout = toIntExact(TimeUnit.MILLISECONDS.convert(startTimeout, timeUnit));
            this.readTimeout = toIntExact(TimeUnit.MILLISECONDS.convert(grabTimeout, timeUnit));
        } else {
            this.connectionTimeout = -1;
            this.readTimeout = -1;
        }
```

**Comment by saudet** [comment]
Thanks for the fix! Could you send that as a pull request?

**Comment by jsichi** [comment]
https://github.com/bytedeco/javacv/pull/415

**Comment by JLLeitschuh** [comment]
My bad, good catch

**Comment by saudet** [comment]
Fix included in JavaCV 1.3. Thanks for the contribution!
